packages/react/src/NavList/NavList.tsx

```diff
@@ -290,6 +291,31 @@ const Group: React.FC<NavListGroupProps> = ({title, children, sx: sxProp = defau
 
 Group.displayName = 'NavList.Group'
 
+export type NavListGroupHeadingProps = ActionListGroupHeadingProps
+
+/**
+ * This is an alternative to the `title` prop on `NavList.Group`.
+ * It was primarily added to allow links in group headings.
+ */
+const GroupHeading: React.FC<NavListGroupHeadingProps> = ({sx: sxProp = defaultSxProp, ...rest}) => {
+  return (
+    <ActionList.GroupHeading
+      as="h3"
+      sx={merge<SxProp['sx']>(
+        {
+          '> a {': {
+            color: 'var(--fgColor-default)',
+            textDecoration: 'inherit',
+            ':hover': {textDecoration: 'underline'},
+          },
+        },
+        sxProp,
+      )}
```

Comment:
Do we still want to use `sx` here?

Reply:
Is it possible to use regular CSS for component styling in PRC yet?

Reply:
If you're adding new styles I think it's safe to go directly to CSS modules. The risk is entirely around the fact that there could be other styles in dotcom relying on the existing sx styling architecture.